src/project-tests/tribute-page-tests.js

```diff
@@ -9,15 +9,15 @@ export default function createTributePageTests() {
     describe("#Tribute Page tests", function() {
 
         describe("#Content", function() {
-            it('1. My tribute page should have a main <div> element with corresponding id="main", which contains ' +
+            it('1. My tribute page should have a main element with corresponding id="main", which contains ' +
```

Comment:
Sounds good, I'll make these changes tomorrow morning.  I got nervous when I saw changes in the build file so I didn't commit them 😄 .